tests/Database/AuthorizeArgsTests/AuthorizeArgsTest.php

```diff
@@ -11,8 +11,6 @@ protected function getEnvironmentSetUp($app): void
     {
         parent::getEnvironmentSetUp($app);
 
-        $app['config']->set('graphql.controllers', GraphQLController::class . '@query');
-
```

Comment:
The config has been renamed, but this actual setting had no impact anyway => removed